tools/base/fpgainfo/board.c

```diff
@@ -400,15 +411,18 @@ fpga_result fpgainfo_board_info(fpga_token token)
 
 	res = load_board_plugin(token, &dl_handle);
 	if (res != FPGA_OK) {
-		OPAE_ERR("Failed to load board plugin\n");
+		OPAE_MSG("Failed to load board plugin\n");
 		goto out;
 	}
 
+	if(dl_handle == NULL)
+		return FPGA_OK;
```

Comment:
Why FPGA_OK if we failed to load the library?

Reply:
removed
fixed